dags/multipod/configs/maxtext_gke_config.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities to construct configs for maxtext DAG on GKE."""
+
+from xlml.apis import gcp_config, metric_config, task, test_config
+from dags import test_owner
+from dags.vm_resource import TpuVersion, Project, ClusterName
+from typing import Iterable
+
+
+def get_maxtext_configs_aot_config(
```

Comment:
Is there anything aot specific about this config? I would like to use it for my convergence tests - could this be named get_maxtext_gke_config instead?

Reply:
Also nothing maxtext specific - could the file be renamed `gke_config` and the method `get_gke_config`?

Reply:
Good call, done!

Reply:
Sorry I already submitted https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/180 - I thought you were going to be out for a week! I renamed the method to `get_maxtext_gke_config`, but I think it would be better if you renamed the file and method to be maxtext-free (but keeping the name the same is fine too). I hope rebasing on top of my PR isn't too painful... I basically rebased on top of this PR except I changed the method name. If you do change the name you will also have to change in the [maxtext_convergence](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/multipod/maxtext_convergence.py) test I added

Reply:
Sounds good! I rebased and changed the name to be maxtext-free